data/json/professions.json

```diff
@@ -4232,5 +4232,65 @@
     "points": -3,
     "items": { "both": [ "shorts", "heroin", "syringe" ], "male": [ "briefs" ], "female": [ "bra", "panties" ] },
     "addictions": [ { "intensity": 40, "type": "opiate" } ]
+  },
+  {
+    "type": "profession",
+    "ident": "yoga_instructor",
+    "name": "Yoga Instructor",
+    "description": "You were in the middle of downward dog. Now you need to get down and out.",
+    "points": 2,
+    "traits": [ "FAST_REFLEXES", "LIGHTEATER", "THINSKIN", "FLIMSY" ],
+    "skills": [ { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "runner_bag", "unitard", "lowtops", "socks_ankle", "wristwatch" ],
+        "entries": [ { "group": "charged_cell_phone" } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "panties" ]
+    },
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "runner",
+    "name": "Runner",
+    "description": "Forest Chump was your role model growing up. Time to make him proud.",
+    "points": 2,
+    "traits": [ "GOODCARDIO", "HUNGER", "FLEET", "FLIMSY" ],
```

Comment:
I can see only GOODCARDIO as justified.

Reply:
The body type of the average runner is slender. Excessive amounts of cardio will put a burden on the body's ability to conserve mass. To offset the increase in metabolism through extended long term activities, the body will begin consuming: stored sugars->stored fats -> stored muscle mass, in that order. Exercising frequently will increase your resting metabolic resting rate. Your caloric intake to maintain the levels of fitness will increase. The same result is true when you increase your body mass(Body building). The larger your body mass, the more calories required to maintain the same level of body mass.

I do agree with you on the flimsy flag however. I misread that and thought it translated to lower weight carried. I mixed it up with bad back. Although it would be true that the average runner is not capable of performing at the same strength levels of a non runner. They don't have enough muscle mass. What I should have done was create a new trait to reflect a %15 decrease in weight carried instead.